### PR TITLE
CIV-2488 - Reject - You've paid less than claim amount

### PR DESCRIPTION
**Pull request by commitBlob**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-2488

### Change description ###
- add full reject - you paid less than claim amount page and appropriate tests
- add language translation object
- refactor language toggle and `govuk-body` class
- fix typos and minor cleanup

Page link: /case/:caseId/response/full-rejection/you-have-paid-less

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/response/rejection/fullReject/youHavePaidLess.controller.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as express from 'express';
```
[the remark this answers is not in the document]
Comment (commitBlob) [reply src/main/routes/features/response/rejection/fullReject/youHavePaidLess.controller.ts]: Have no idea why I've done .controller, must be autopilot 😂 will change now

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -10,6 +10,19 @@
   "View": "Weiv",
   "Save and continue": "Arbed a pharhau",
   "After you have completed all the actions you will be taken to a page where you can check your answers before submitting.": "Ar ôl i chi gwblhau'r holl gamau gweithredu byddwch yn cael eich tywys i dudalen lle gallwch wirio'ch atebion cyn eu cyflwyno.",
+  "COMMON": {
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:13]: I like this approach, to have a key with COMMON, PAGES, etc, Maybe we all should agree with that before start merging. I saw in Tanya's PR that she is not using "PAGES" key and in my example I didn't use it neither. But I'm happy to use your way.

File: src/main/routes/urls.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ const BASE_CASE_URL = '/case/:id';
 const BASE_CASE_RESPONSE_URL = `${BASE_CASE_URL}/response`;
 const STATEMENT_OF_MEANS_URL = `${BASE_CASE_RESPONSE_URL}/statement-of-means`;
 const MEDIATION_URL = `${BASE_CASE_URL}/mediation`;
+const FULL_REJECTION = `${BASE_CASE_RESPONSE_URL}/full-rejection`;
```
Comment (taomoh) [inline src/main/routes/urls.ts:5]: append _URL

File: src/main/routes/urls.ts [hunk 4]
```diff
@@ -66,3 +67,4 @@ export const IMPACT_OF_DISPUTE_URL = `${BASE_CASE_RESPONSE_URL}/impact-of-disput
 export const CITIZEN_REJECT_ALL_CLAIM_URL = `${BASE_CASE_RESPONSE_URL}/reject-all-of-claim`;
 export const CITIZEN_CONTACT_THEM_URL = `${DASHBOARD_URL}/:id/contact-them`;
 export const RESPONSE_CHECK_ANSWERS_URL = `${BASE_CASE_RESPONSE_URL}/check-and-send`;
+export const CITIZEN_FULL_REJECTION_YOU_PAID_LESS = `${FULL_REJECTION}/you-have-paid-less`;
```
Comment (taomoh) [inline src/main/routes/urls.ts:70]: append _URL

**Review by taomoh — COMMENTED** [review]
LG. some minor comments?

File: src/main/routes/urls.ts (hunk 4 above)
Comment (commitBlob) [reply src/main/routes/urls.ts]: Good spot, should be pushed up now.
Thanks

**Review by laarmada — APPROVED** [review]
good job!

**Review by taomoh — APPROVED** [review]

**Pull request merged**